lib/phobos_db_checkpoint/cli.rb

```diff
@@ -34,23 +34,35 @@ def init
              aliases: ['-d'],
              default: 'db/migrate',
              banner: 'Destination folder relative to your project'
+      option :config,
+             aliases: ['-c'],
+             default: 'config/database.yml',
+             banner: 'Database configuration relative to your project'
       def copy_migrations
+        if options[:config]
+          ENV['DB_CONFIG'] = options[:config]
+        end
+
+        unless (ActiveRecord::Base.connection_pool.with_connection { |con| con.active? } rescue false)
```

Comment:
I think this line is doing a lot.
It's an `unless`, with a block, and a `rescue false`...
Let's extract it to a method so it's easier to read